js/background/inject/inject.js

```diff
@@ -103,8 +103,7 @@ $j(document).ready(function () {
                 showPasswordPicker(form);
             }
         }
-
-        // $j(el).bind('click', onClick);
+        $j(el).unbind('click');
```

Comment:
You should give it the onClick as [parameter](http://api.jquery.com/unbind/).
Because this code will unbind every onclick event on the element, even those from the website self.